src/Dash.NET/DashComponents/CoreComponents/Store.fs

```diff
@@ -27,14 +27,14 @@ module Store =
     type Prop =
         | ClearData         of bool
         | Data              of string
-        | ModifiedTimestamp of int
+        | ModifiedTimestamp of int64
         | StorageType       of PersistenceTypeOptions
         static member toDynamicMemberDef (prop:Prop) =
             match prop with
             | ClearData         p-> "clear_data"        , box p
             | Data              p-> "data"              , box p
             | ModifiedTimestamp p-> "modified_timestamp", box p
-            | StorageType       p-> "storage_type"      , box p
+            | StorageType       p-> "storage_type"      , p |> PersistenceTypeOptions.convert |> box
```

Comment:
the pipe into `box` should be unnecessary, as `convert` already returns a boxed value.

Reply:
Thanks @kMutagene - I pushed the update